supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/AwsBatchInitializationActor.scala

```diff
@@ -69,6 +69,18 @@ class AwsBatchInitializationActor(params: AwsBatchInitializationActorParams)
   private val configuration = params.configuration
   override implicit val system = context.system
 
+  override def beforeAll(): Future[Option[BackendInitializationData]] = {
+    configuration.fileSystem match {
+     case AWSBatchStorageSystems.s3  => super.beforeAll
+     case _ => {
+                   initializationData map { data =>
+                   publishWorkflowRoot(data.workflowPaths.workflowRoot.pathAsString)
+                   Option(data)
```

Comment:
A similar question about formatting, including whether the extra block inside the `case` statement is necessary?